login/service_whitebox_test.go

```diff
@@ -156,8 +156,7 @@ func TestEncodeTokenOK(t *testing.T) {
 	refreshToken := "refreshToken%@!/\\&?"
 	tokenType := "tokenType%@!/\\&?"
 	expiresIn := 1800
-	var refreshExpiresIn float64
-	refreshExpiresIn = 2.59e6
+	refreshExpiresIn := 2.59e6
```

Comment:
As far as I understand if we don't specify float64 then both float64 and float32 may be used. It will be platform specific. But these tests should use float64. Otherwise they may fail on 32-bit systems. I didn't test them on a 32-bit system though.